ansible-completion.bash

```diff
@@ -33,11 +33,11 @@ _ansible_complete_host() {
     # search in the ansible.cfg for a hostfile entry
     if [ -z "$inventory_file" ]; then
         [ -f /etc/ansible/ansible.cfg ] && inventory_file=$(awk \
-            '/^(hostfile|inventory)/{ print $3 }' /etc/ansible/ansible.cfg)
+            '/^(hostfile =|inventory =)/{ print $3 }' /etc/ansible/ansible.cfg)
```

Comment:
What about allowing an arbitrary amount of whitespace before and after? That should work for everyone.

```suggestion
            '/^(hostfile[[:space:]]*=[[:space:]]*|inventory[[:space:]]*=[[:space:]]*)/{ print $3 }' /etc/ansible/ansible.cfg)
```